cachelib/allocator/CacheAllocator-inl.h

```diff
@@ -372,7 +403,8 @@ CacheAllocator<CacheTrait>::allocateInternalTier(TierId tid,
                                              typename Item::Key key,
                                              uint32_t size,
                                              uint32_t creationTime,
-                                             uint32_t expiryTime) {
+                                             uint32_t expiryTime,
+                                             bool fromEvictorThread) {
```

Comment:
Does `fromEvictorThread` also set to `true` when it is called from promoter thread? If so, let´s rename variable to have more generic name, e.g. `fromBgThread`?